src/main/resources/layout.vm

```diff
@@ -24,7 +24,9 @@
         <link rel="shortcut icon" href="#include("/static/favicon.base64")">
     </head>
     <body data-user-id="$user">
-        #include("/static/gtm.html")
+        #if($gtmId != "")
```

Comment:
Ah, this is why you make it an empty String.

Reply:
Yeah, I originally wanted to check for null but TemplateUtil#model doesn't allow for null values so I went with the next best thing - an empty string

Reply:
> TemplateUtil#model doesn't allow for null values

Thanks, that's a bug in Javalin then. Will fix.